platforms/ide/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r86/ProblemProgressEventCrossVersionTest.groovy

```diff
@@ -165,39 +157,29 @@ class ProblemProgressEventCrossVersionTest extends ToolingApiSpecification {
         ''                         | null            | ''                                          | null
     }
 
+    def runTask(){
```

Comment:
💅 There are a few cases in this PR where the formatting is slightly off. There's no space between the method closing parenthesis and the opening curly brace. A quick `Reformat code` action on the changed files would be useful.

```suggestion
    def runTask() {
```